### PR TITLE
validate addresses from qr

### PR DESCRIPTION
**Pull request by salomegeo**
## Description
799

## Change Type
- Bug Fix

## Testing Steps
Detail the steps required for the reviewer(s) to verify and test these changes.

## PR Creator Checklist
- [ ] Code compiles correctly
- [ ] No lint issues exist (verified via `yarn lint`)
- [ ] All unit tests pass (verified via `yarn test`)
- [ ] Updated `README.md` and other documentation (if necessary)

## PR Reviewer Checklist
- [x] Change validated and application spot checked
- [x] Code styles and best practices met
- [x] Code is readable and commented when necessary

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ import bip21 from 'bip21'
 
 import { actions } from 'data'
 import QRCodeCapture from './template.js'
+import * as C from 'services/AlertService'
+import {utils} from '../../../../blockchain-wallet-v4/src'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js]: lets not use relative paths. should be `import {utils} from 'blockchain-wallet-v4/src'`

File: packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js [hunk 2]
```diff
@@ -27,31 +29,51 @@ class QRCodeCaptureContainer extends React.PureComponent {
...
-    this.props.updateUI({ btcAddress: { toggled: false } })
+    try {
+      const { address, options } = bip21.decode(data)
+      const { amount, message } = options
+      this.props.formActions.change('sendBtc', 'to', address)
+      this.props.formActions.change('sendBtc', 'amount', amount)
+      this.props.formActions.change('sendBtc', 'message', message)
+      this.props.updateUI({ btcAddress: { toggled: false } })
+    } catch (e) {
+      this.props.alertActions.displayError(C.BTC_ADDRESS_INVALID)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js]: Are we sure that the only thing that could cause issue is an invalid address?

Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js]: Same comment goes for the other scan type methods as well.

Comment (salomegeo) [reply packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js]: I kinda am, like the rest is UI change.

File: packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js (hunk 1 above)
Comment (salomegeo) [reply packages/blockchain-wallet-v4-frontend/src/components/QRCodeCapture/index.js]: done

**Pull request merged**